sdk/stdlib-sys/src/stdlib/crypto/hashes.rs

```diff
@@ -174,3 +192,41 @@ pub fn sha256_hash_1to1(input: [u8; 32]) -> [u8; 32] {
 pub fn sha256_hash_2to1(input: [u8; 64]) -> [u8; 32] {
     hash_2to1(input, extern_sha256_hash_2to1)
 }
+
+/// Computes the hash of a sequence of field elements using the Rescue Prime Optimized (RPO)
+/// hash function.
+///
+/// This maps to the `std::crypto::rpo::hash_memory` procedure in the Miden stdlib.
+///
+/// # Arguments
+/// * `elements` - A slice of field elements to be hashed
+#[inline]
+pub fn hash_elements(elements: &[Felt]) -> Digest {
```

Comment:
@bitwalker Since Miden stdlib `hash_memory` which we are calling underneath expects a word-aligned pointer to the felts, we need to make sure that `elements` slice is aligned. See my implementation below. It's compiled down to Wasm `memory.copy` which is a byte copy and besides being hugely expensive copying byte by byte it does not work with felts - see https://github.com/0xMiden/compiler/issues/594 

I'm thinking of switching to the `pub fn hash_elements(elements: Vec<Felt>) -> Digest` signature. Since our BumpAlloc is word-aligned, we can assume that `elements` is word-aligned and get away with an assert.